src/options/bv_options.toml

```diff
@@ -2,6 +2,17 @@ id     = "BV"
 name   = "Bitvector theory"
 header = "options/bv_options.h"
 
+[[option]]
+  name       = "bvProofFormat"
+  category   = "expert"
```

Comment:
I'd argue that expert level is appropriate since it is something internal. If a user is using `--check-proofs`, the user never sees the internal representation of the unsat proof, so it shouldn't really matter to them.

Reply:
Whoops, the :+1: was in response to @4tXJ7f comment, not mine.